pkg/alertmanager/alertmanager.go

```diff
@@ -16,11 +16,13 @@ import (
 	"github.com/go-kit/log"
 	"github.com/go-kit/log/level"
 	"github.com/pkg/errors"
+	"github.com/prometheus/alertmanager/alertobserver"
 	"github.com/prometheus/alertmanager/api"
 	"github.com/prometheus/alertmanager/cluster"
 	"github.com/prometheus/alertmanager/cluster/clusterpb"
 	"github.com/prometheus/alertmanager/config"
 	"github.com/prometheus/alertmanager/dispatch"
+	"github.com/prometheus/alertmanager/featurecontrol"
```

Comment:
nit: is this belong to this change?

Reply:
This is needed to support the latest changes in AM. This change https://github.com/prometheus/alertmanager/pull/3045 adds a featurecontrol parameter to `NewPipelineBuilder`.